mava/callbacks/base.py

```diff
@@ -259,6 +259,19 @@ def on_execution_update_end(self, executor: SystemExecutor) -> None:
         """End of updating executor parameters."""
         pass
 
+    # FORCE UPDATE
+    def on_execution_force_update_start(self, executor: SystemExecutor) -> None:
```

Comment:
Why do we need these new hooks?

Reply:
We need it to force the immediate update of the executor values when we finish the evaluation (`mava/environment_loop.py` line 648)